dpnp/random/dpnp_iface_random.py

```diff
@@ -496,6 +437,65 @@ def gamma(shape, scale=1.0, size=None):
     return call_origin(numpy.random.gamma, shape, scale, size)
 
 
+def geometric(p, size=None):
```

Comment:
it is not good idea to mix "refactoring" with "development".
No changes needed. Please separate such things in future.

Reply:
There is no "development" here. I just changed places of some functions. Just Github is going crazy. There is only refactoring here. The refactoring idea is reflected in the PR title.